tests/Microsoft.DotNet.Framework.Docker.Tests/ImageTests.cs

```diff
@@ -121,6 +136,39 @@ private void VerifyImages(ImageDescriptor imageDescriptor, string appDescriptor,
             }
         }
 
+        private void VerifyWcfImages(ImageDescriptor imageDescriptor, string appDescriptor, string runCommand, bool includeRuntime)
+        {
+            List<string> appBuildArgs = new List<string> {  };
+            if (includeRuntime)
+            {
+                string baseRuntimeImage = GetImage("wcf", imageDescriptor.RuntimeVersion, imageDescriptor.OsVariant);
+                appBuildArgs.Add($"BASE_RUNTIME_IMAGE={baseRuntimeImage}");
+            }
+
+            string appId = $"{appDescriptor}-{DateTime.Now.ToFileTime()}";
+            string workDir = Path.Combine(
+                Directory.GetCurrentDirectory(),
+                "projects",
+                $"{appDescriptor}-{imageDescriptor.RuntimeVersion}");
+
+            try
+            {
+                DockerHelper.Build(
+                    tag: appId,
+                    dockerfile: Path.Combine(workDir, "Dockerfile"),
+                    buildContextPath: workDir,
+                    buildArgs: appBuildArgs);
+
+                DockerHelper.Run(image: appId, containerName: appId, command: runCommand, web: true);
+                DockerHelper.VerifyHttpResponseFromContainer(appId, "/Service1.svc");
+            }
+            finally
+            {
+                DockerHelper.Stop(containerName: appId);
+                DockerHelper.DeleteImage(appId);
+            }
+        }
```

Comment:
After looking at this method I see a lot of duplicated code with the VerifyFxImages.  Should a common method be created that is shared across WCF, ASP, and the runtime?

```
private void VerifyImages(
    ImageDescriptor imageDescriptor,
    IEnumerable<string> buildArgs,
    string appDescriptor,
    string runCommand,
    string serviceUrl)
{
    string appId = $"{appDescriptor}-{DateTime.Now.ToFileTime()}";
    string workDir = Path.Combine(
        Directory.GetCurrentDirectory(),
        "projects",
        $"{appDescriptor}-{imageDescriptor.RuntimeVersion}");

    try
    {
        DockerHelper.Build(
            tag: appId,
            dockerfile: Path.Combine(workDir, "Dockerfile"),
            buildContextPath: workDir,
            buildArgs: buildArgs);

        DockerHelper.Run(image: appId, containerName: appId, command: runCommand, detach: !string.IsNullOrEmpty(serviceUrl));

        if (isWeb)
        {
            DockerHelper.VerifyHttpResponseFromContainer(appId, serviceUrl);
        }
    }
    finally
    {
        DockerHelper.DeleteContainer(appId);
        DockerHelper.DeleteImage(appId);
    }
}
```

Reply:
Fair enough, I will update this.